services/visual_recognition/v1.js

```diff
@@ -0,0 +1,237 @@
+module.exports = function(RED) {
+
+	var cfEnv = require("cfenv");
+
+	var watson = require('watson-developer-cloud');
+	var temp = require('temp');
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 2 space characters but found 0.](https://www.codacy.com/app/rezgui-y/node-red-node-watson/file/2400711164/issues/source?bid=3225115&fileBranchId=3232279#l6)